core/src/main/java/org/apache/calcite/sql/fun/SqlLibraryOperators.java

```diff
@@ -2110,6 +2115,7 @@ private SqlLibraryOperators() {
           OperandTypes.VARIADIC,
           SqlFunctionCategory.SYSTEM);
 
+  @LibraryOperator(libraries = {ORACLE})
```

Comment:
Why do we need this?
As discussed please verify in master branch if all TC are passing without the change.

Reply:
In master some tests were failing with exception
com.google.common.util.concurrent.ExecutionError: java.lang.AssertionError: Operator must have annotation: IN_STRING
which was due to @LibraryOperator annotation missing for IN_STRING. It was removed by someone accidently.
I wasnt able to run or debug one related test case.

So, to fix that i added this annotaion back.